js/index.js

```diff
@@ -0,0 +1,79 @@
+const opneNave = () => {
+  document.getElementById('mobileNave').style.width = '100%';
+};
+const closeNave = () => {
+  document.getElementById('mobileNave').style.width = '0';
+};
+
+opneNave();
+closeNave();
+
+// _______________Dinamic Content___________________
+
+const artistsData = [
+  {
+    name: 'Shepard Fairey',
```

Comment:
- [ ] your speakers section is missing the title including the small red line below it. .. Please add that
-  [optional] please kindly reduce the gray line under the speaker title, so it matches the original version.
yours
![Screenshot (1308)_LI](https://github.com/Fernando426/CapstoneN01/assets/65298048/beeaa385-37c3-4197-9230-1cf7376ed549)
original
![Screenshot (1292)](https://github.com/Fernando426/CapstoneN01/assets/65298048/79ef41ab-ac3f-494c-853d-3d3149a0e454)